.github/workflows/rust.yml

```diff
@@ -39,11 +39,16 @@ jobs:
     - name: Run tests
       run: |
         pushd borsh
+        echo 'cargo test --no-run ...'
         cargo test --no-run
+        echo 'cargo test ...'
         cargo test
-        cargo test --no-default-features
-        cargo test --all-features
+        echo 'cargo test --no-default-features --features external-hashcollections,rc ...'
```

Comment:
Also add `cargo test --no-default-features` and fix compilation errors (I expect it to fail as there are traits being implemented for HashMap / HashSet)